.github/workflows/images.yaml

```diff
@@ -0,0 +1,107 @@
+name: Build
+
+on:
+  push:
+    tags:
+      - '*.*.*'
```

Comment:
For the tags, it needs to follow the same logic that Kenyi implemented on the PyPi action.

Reply:
I added missing `reqmgr2`, `workqueue` and `t0_reqmon`. The corresponding areas for docker files is also added in CMSKubernetes.

Reply:
Now the build tag is fixed too.